From fa94d4d92a5dabac28b4b12aec4764abbc2a7859 Mon Sep 17 00:00:00 2001
From: Artem Polishchuk <tim77@users.noreply.github.com>
Date: Sun, 22 Mar 2020 14:34:09 +0200
Subject: [PATCH] Add include which requires new GCC 10

Fix #22
---
 src/section.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/section.cpp b/src/section.cpp
index 79be269..887a8a3 100644
--- a/src/section.cpp
+++ b/src/section.cpp
@@ -1,5 +1,6 @@
 #include <wayfire/config/section.hpp>
 #include <map>
+#include <stdexcept>
 
 struct wf::config::section_t::impl
 {
